en/getting-started.md

```diff
@@ -1,10 +1,104 @@
 ---
-title: Color Palettes - Design System
-_description: The Avatar Component Symbol is a graphical representation of personal information. 
-_keywords: Design System, Sketch, Ignite UI for Angular, UI Library, Colors, Palettes
+title: Getting Started - Design System
+_description: How to set up the Indigo Design Libraries and craft beautiful digital experiences with Sketch. 
+_keywords: Design System, Sketch, Ignite UI for Angular, UI Library, Getting Started, Tutorial
 ---
 
-- Download libraries
-- Import libraries in Sketch
-- Create minimal artboard in 5 steps
-- Explain override selection and why you shouldn't just pick anything but only truly overridable things
+## Getting Started
+
+This topic describes the process of downloading and adding the Indigo Design Libraries into Sketch and how to craft your first design using and customizing the Components necessary for building a registration screen.
+
+### Downloading and Adding the Libraries in Sketch
+
+First you will need to [Download the Indigo Design Sketch Libraries](TODO-add the link) to yours drive. You will get three Sketch files:
```

Comment:
First, you will need to [Download the Indigo Design Sketch Libraries](TODO-add the link) to your drive.

Reply:
fixed